discussions/api.py

```diff
@@ -0,0 +1,115 @@
+"""API for open discussions integration"""
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.db import transaction
+from open_discussions_api.client import OpenDiscussionsApi
+from open_discussions_api.constants import ROLE_STAFF
+
+from discussions.models import DiscussionUser
+from discussions.exceptions import DiscussionUserSyncException
+
+
+def get_staff_client():
+    """
+    Gets a client configured for user management
+    """
+    if not settings.OPEN_DISCUSSIONS_JWT_SECRET:
```

Comment:
I'm not certain this should be a required setting for MicroMasters, particularly since this code is likely to be deployed prior to production being configured for it. Also, most of our `ImproperlyConfigured` usages to date are raised when the code using the setting is run.